controllers/tasks.js

```diff
@@ -177,7 +178,12 @@ const updateTask = async (req, res) => {
     if (!task.taskData) {
       return res.boom.notFound("Task not found");
     }
-
+    if (req.body.assignee) {
```

Comment:
are we sure if we are getting the `assignee` field, if not I would recommend use optional chaining (i.e `?` operator) in the `if` block as it would avoid unexpected error.